src/components/checkbox/partial-styles/_helper-text.scss

```diff
@@ -2,6 +2,9 @@
 
 .limel-checkbox-helper-line {
     @include shared_input-select-picker.looks-like-helper-line;
+    grid-column-start: 1;
+    grid-column-end: -1;
+    padding-left: var(--width-of-checkbox);
```

Comment:
Sorry @LucyChyzhova . It's my fault telling you to have this padding. But now I don't think we need this actually, and remember why it was like that before.

The reasons is that when checkbox is in a form, the helper label looks nicer if it's aligned with other helper texts of other components. 👇 

![image](https://user-images.githubusercontent.com/35954987/152325032-b0948392-6e69-436f-b86a-9ca0ab907cd1.png)